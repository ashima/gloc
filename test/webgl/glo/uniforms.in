uniform vec3 pos;

vec3 get_pos() { return pos; }
