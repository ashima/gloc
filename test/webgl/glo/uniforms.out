{
  "glo": [ 0, 1, 0 ],
  "target": [ "webgl", [ 1, 0, 0 ] ],
  "meta": {
    "copyright": [ "", 2012, "" ],
    "author": {},
    "license": null,
    "library": null,
    "version": null,
    "build": null
  },
  "units": [
    {
      "insym": [],
      "outsym": [ "s", "pos", "get_pos", "S" ],
      "inmac": [],
      "opmac": [],
      "outmac": [],
      "source":
        "#line 1 GLOC_0\nuniform vec3 pos;\nuniform struct S { int k; } s;\n\nvec3 get_pos() { return pos; }"
    }
  ],
  "linkmap": { "0": "<stdin>#u=0" }
}
